src/library/filescanner.c

```diff
@@ -1989,6 +2087,9 @@ queue_save(const char *virtual_path)
 	  continue;
 	}
 
+      if (queue_item.data_kind == DATA_KIND_HTTP)
```

Comment:
Queue items that are not in the files table have the magic file_id of 9999999 (see `library_add_queue_item`). Would be good to only add special handling for those items. Maybe instead of the magic value add a constant to db.h for it.